File: Exercicios em Java/src/Exercicio3.java
import java.util.Scanner;

public class Exercicio3{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num = 0;
        int cont = 0;
        int pares = 0;

        while (cont < 10) {
            System.out.println("Digite um número: ");
            num = sc.nextInt();
            if (num % 2  == 0 ){
                pares++;
            }
            cont++;
       }
       System.out.println(pares+" números são pares.");
    }
}

File: Exercicios em Python/Exercicio3.py
#3. Implemente um programa que solicite 10 números inteiros ao usuário. Informe quantos números são pares.
pares = 0
cont = 0

while cont < 10:
    num = input("Informe um número: ")
    numInt = int(num)
    cont +=1
    if numInt % 2 == 0:
        pares += 1 
    
print("Foram informados ",pares, " numeros pares.")